py3status/modules/keyboard_layout.py

```diff
@@ -102,6 +137,19 @@ def _setxkbmap(self):
         led_mask = re.match(LEDMASK_RE, xset_output).groups(0)[0]
         return layouts[int(led_mask)]
 
+    def _set_active(self, delta):
+        self._active += delta
+        self._active = self._active % len(self._layouts)
+        layout = self._layouts[self._active]
+        self.py3.command_run(['setxkbmap', '-layout', layout])
```

Comment:
Cool, but what if I have `xkblayout-state` installed? This goes off the path (slightly) as we would read via `xkblayout-state` and set via `setxkbmap`. Ideally, we strives to do it cleanly. Also, Idc.

Reply:
> what if I have xkblayout-state installed? 

good question I don't have it so no ideas :)

Reply:
Setting should be as [easy](https://github.com/nonpop/xkblayout-state#setting-the-active-keyboard-layout) as `xkblayout-state set <+1/-1>` (use `delta`), but if this takes time to implement/test, I'd prefer to see this merged as it is and improved separately 🙂